app/src/main/java/com/nimbl3/ui/main/MainViewModel.kt

```diff
@@ -10,11 +11,10 @@ import com.nimbl3.ui.main.data.Data
 import io.reactivex.Observable
 import io.reactivex.subjects.BehaviorSubject
 import io.reactivex.subjects.PublishSubject
-import javax.inject.Inject
 
 class MainViewModel
-@Inject constructor(private val repository: ApiRepository,
-                    private val schedulers: SchedulersProvider) : BaseViewModel(), Inputs, Outputs {
+@ViewModelInject constructor(private val repository: ApiRepository,
+                             private val schedulers: SchedulersProvider) : BaseViewModel(), Inputs, Outputs {
```

Comment:
should we consider to reformat this block to
```
class MainViewModel @ViewModelInject constructor(
    private val repository: ApiRepository,
    private val schedulers: SchedulersProvider
) : BaseViewModel(), Inputs, Outputs {
```
🤔

Reply:
got it, let me refactor it